test/position/sync/chooseOneResults.test.js

```diff
@@ -0,0 +1,90 @@
+const test = require('tape')
+const ChooseOne = require('../../../position/sync/chooseOne')
+const ChooseOnePoll = require('../../../poll/sync/chooseOne')
+const Position = require('../../../position/sync/position')
+const chooseOneResults = require('../../../position/sync/chooseOneResults')
+
+const pietId = '@Mq8D3YC6VdErKQzV3oi2oK5hHSoIwR0hUQr4M46wr/0=.ed25519'
+const mixId = '@Mq8D3YC6VdErKQzV3oi2oK5hHSoIwR0hUQr4M46wr/1=.ed25519'
+const mikeyId = '@Mq8D3YC6VdErKQzV3oi2oK5hHSoIwR0hUQr4M46wr/2=.ed25519'
+const timmyId = '@Mq8D3YC6VdErKQzV3oi2oK5hHSoIwR0hUQr4M46wr/3=.ed25519'
+const tommyId = '@Mq8D3YC6VdErKQzV3oi2oK5hHSoIwR0hUQr4M46wr/4=.ed25519'
+const sallyId = '@Mq8D3YC6VdErKQzV3oi2oK5hHSoIwR0hUQr4M46wr/5=.ed25519'
```

Comment:
nice. I reckon for future ones we could edit the start to make it easy to read output:
```js
const pietId = '@Piet3YC6VdErKQzV3oi2oK5hHSoIwR0hUQr4M46wr/0=.ed25519'
const mixId = '@MixD3YC6VdErKQzV3oi2oK5hHSoIwR0hUQr4M46wr/1=.ed25519'
```